src/postgres/commands/connectPostgresDatabase.ts

```diff
@@ -21,9 +21,16 @@ export async function connectPostgresDatabase(context: IActionContext, treeItem?
     const oldTreeItemId: string | undefined = ext.connectedPostgresDB && ext.connectedPostgresDB.fullId;
     void ext.context.globalState.update(connectedPostgresKey, treeItem.fullId);
     ext.connectedPostgresDB = treeItem;
+    const username: string | undefined = (await treeItem.parent.getFullConnectionString()).username;
+    const serverId: string = treeItem.parent.id;
+    let serverName: string | undefined = serverId;
+    if (username) {
+        serverName = username.includes(serverId) ? username : username + `@${serverId}`;
```

Comment:
So, I don't know about your Postgres server, but my `serverId` includes my entire subscription name and id so it ends up looking like this:

![image](https://user-images.githubusercontent.com/5290572/127551942-a4341091-fffc-4397-a2ea-3757f00a51eb.png)

We definitely don't wanna be including that in their database name 😅

Reply:
oh right, good point! Attached accounts behaves differently so I assumed serverId, but I think label() value of the Server might be more appropriate.